src/databricks/labs/ucx/hive_metastore/tables.scala

```diff
@@ -52,9 +53,19 @@ def metadataForAllTables(databases: Seq[String], queue: ConcurrentLinkedQueue[Ta
   }).toList.toDF
 }
 
-dbutils.widgets.text("inventory_database", "ucx")
-val inventoryDatabase = dbutils.widgets.get("inventory_database")
+def getInventoryDatabase(): String={
+  dbutils.widgets.text("config", "/Workspace/Users/hari.selvarajan@databricks.com/.ucx/config.yml")
```

Comment:
```suggestion
  dbutils.widgets.text("config", "")
```
don't hardcode your username, add a check for config not being empty